### DISCUSSION TITLE
How are you assigning taxonomy to your eDNA metabarcoding sequences?

### DISCUSSION THREAD

**Discussion by ksilnoaa** [NOAA-Omics/NOAA-Omics#12] [3-Bioinformatics]
For those assigning taxonomy to eDNA metabarcoding sequences:

1.  What method are you using? What software?
2. What taxa are you working with?
3. What database(s) are you using?
4. Do you do anything to validate taxa assignments? E.g., check for nonnative species, parameter evaluations, compare with external data...

**Comment by lukenoaa** [comment]
My comment here. Here's a link to the [org page](https://github.com/NOAA-Omics). This is related to #2. Commit ID [e02290c](https://github.com/NOAA-Omics/noaa-omics-templates/commit/e02290cced27bde6b6f43efce37e5d94dfd5b723).

**Comment by marinednadude** [comment]
Historically I have predominantly used the Bayesian Lowest Common Ancestor classifier from [Anacapa Toolkit](https://github.com/limey-bean/Anacapa) 

Our team is actively comparing [REVAMP](https://github.com/McAllister-NOAA/REVAMP) and [Tourmaline](https://github.com/aomlomics/tourmaline).

We've talked about using [Tronko](https://github.com/lpipes/tronko) for quite some time ...
